microsoft-edge/webview2/concepts/data_privacy.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Data & Privacy in WebView2
+description: Overview of WebView2 Data Collection and Practices. 
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/03/2023
+---
+# Data & Privacy in WebView2 
+
+## Background
+The WebView2 team tracks data to help understand and prioritize our customers' requirements, diagnose and debug issues, and overall improve WebView2 as a product.
+Developers consent to this data collection in the WebView2 Runtime Terms and Conditions License, or in the Windows license. Additionally, we follow the standards outlined in the [Microsoft Edge Privacy Whitepaper](https://learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/).

```

Comment:
@liminzhu the addition of "or in the Windows license" was in the original doc that Jason had started, do we still need this part or is referencing the WebView2 license sufficient?